Feat: PT_BR localization

PrincessStellar opened the pull request.

PrincessStellar committed:
Create pt_br

Dragon-Seeker commented:
@PrincessStelllar Hey, thanks for the PR, but with some talks we like to get some more eyes on this PR as I want to make sure its without big errors or issues, as written in the new [translation info](https://github.com/wisp-forest/accessories/blob/1.21.x/info/INFO_ABOUT_TRANSLATIONS.md) with requiring two or more people to review the PR.

Jmpp2909 commented:
Howdy @Dragon-Seeker ,
I'm one of the members of the Dev team over at [AllTheMods](https://github.com/AllTheMods).
Stellar has done translations for our packs and associated mods, and if you check her profile, for multiple other mods and modpacks too.
I personally reviewed her translations and can vouch that they are very high quality.

If you'd like, I can review these myself too. (I'm native Portuguese too)

PrincessStellar committed:
Updated PT_BR localization

PrincessStellar committed:
Updated PT_BR localization

- The translation file has been reverted because I checked and made an update thinking it was for version 1.21.1, but it was actually for a new version of Minecraft. I have changed some translations.

PrincessStellar committed:
Updated PT_BR localization
